step.yml

```diff
@@ -238,7 +276,7 @@ outputs:
     opts:
       title: Map of filenames and Permanent Download URLs
       description: |-
-        This input creates a permanent Download URL for artifacts. The URL can be shared in any communication channel and it will not expire.
+        This input creates a permanent Download URL for all artifacts. The URL can be shared in any communication channel and it will not expire.
```

Comment:
This is an output of the step so the `This input` part might be confusing. maybe `This output contains a permanent...` would be more clear